pyseir/deployment/webui_data_adaptor_v1.py

```diff
@@ -176,9 +219,13 @@ def map_fips(self, fips):
             output_model[schema.INFECTED_B] = hosp_rescaling_factor * np.interp(
                 t_list_downsampled, t_list, output_for_policy["HGen"]["ci_50"]
             )  # Hosp General
-            output_model[schema.INFECTED_C] = icu_rescaling_factor * np.interp(
-                t_list_downsampled, t_list, output_for_policy["HICU"]["ci_50"]
-            )  # Hosp ICU
+
```

Comment:
All these conversions should be done on the pyseir side somewhere. There's a lot of scaling/math happening in a output mapper script. Apply icu_rescaling_fator upstream somewhere closer to the creation point (where the context of what is going on is clearer). Probably a different PR.

Reply:
I'm going to save this for another PR, but I completely agree. This conversion needs to live much closer to the model -- the interpolation and integer conversion too, IMO.